### PR TITLE
Display error message when there are no test files

### LINKED ISSUE #55: Crash when setting an non-existent path
```
$ ava does-not-exist
```

### PR DESCRIPTION
**Pull request by vdemedes**
Fixes #55 

Output:

screen shot 2015-09-23 at 9 21 37 pm

### CODE REVIEW

File: cli.js [hunk 1]
```diff
@@ -106,6 +106,12 @@ function init(files) {
 			return path.join(process.cwd(), file);
 		})
 		.then(function (files) {
+			if (files.length === 0) {
+				log.error('Couldn\'t find any files to test');
```
Comment (sindresorhus) [inline cli.js]: `log.error('Couldn\'t find any files to test\n');` instead of the extract `log.write()` call.

File: cli.js [hunk 2]
```diff
@@ -106,6 +106,11 @@ function init(files) {
 			return path.join(process.cwd(), file);
 		})
 		.then(function (files) {
+			if (files.length === 0) {
```
Comment (Qix-) [inline cli.js]: @sindresorhus don't we go for `!files.length`? Or am I just thinking back to my C days?

Comment (sindresorhus) [inline cli.js]: I prefer to be explicit.

Comment (Qix-) [inline cli.js]: Fair enough.

**Pull request merged**